omf-python/src/file/reader.rs

```diff
@@ -388,7 +403,7 @@ impl PyReader {
         }
     }
 
-    /// Read bytes of an Image.
+    /// Read bytes of an Image and return a bytes array.
```

Comment:
Could be more descriptive.
```suggestion
    /// Read the encoded bytes of an Image and return them as a bytes array. The bytes
    /// may be in JPEG or PNG format; use the image header to distinguish the two.
```